focus-ios/focus-ios-tests/FullFunctionalTests.xctestplan

```diff
@@ -28,6 +28,7 @@
         "BrowsingTest\/testActivityMenuRequestDesktopItem()",
         "CopyPasteTest\/testClipboard()",
         "CopyPasteTest\/testPastenGo()",
+        "DragAndDropTest",
```

Comment:
Should we remove DragAndDropTest from the repo and TestRail? It fails often but passes for me. :(

Reply:
let's try to improve them at some point instead of removing them.